scripts/windows_build_wheels.py

```diff
@@ -219,11 +219,20 @@ def fixup_wheels():
         fixup_wheel(wheel)
 
 
-def test_wheels(single_wheel=False):
+def test_wheels(python_version):
+    python_executable, \
+            python_include_dir, \
+            python_library, \
+            pip, \
+            ninja_executable, \
+            path = venv_paths(python_version)
```

Comment:
Nitpick, to avoid the line continuation trick, the following could be done i think:

```python
(
        python_executable,
        python_include_dir,
        python_library,
        pip,
        ninja_executable,
        path
) = venv_paths(python_version)
```

Reply:
Stylishish! Done.